packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -736,11 +736,11 @@ export async function cleanupTransformationJob() {
 }
 
 export async function resetDebugArtifacts() {
-    const buildLogPath = transformByQState.getPreBuildLogFilePath()
-    if (buildLogPath && fs.existsSync(buildLogPath)) {
-        fs.rmSync(path.dirname(transformByQState.getPreBuildLogFilePath()), { recursive: true, force: true })
+    const buildLogDir = path.join(os.tmpdir(), 'q-transformation-build-logs')
```

Comment:
should this use `makeTemporaryToolkitFolder` ? `os.tmpdir()` can be problematic, as hinted in the comment in `tempDirPath`. https://github.com/aws/aws-toolkit-vscode/blob/56b1c4b785dfec0e4949776f39fa104e2fbca4aa/packages/core/src/shared/filesystemUtilities.ts#L15-L18